src/flusher.h

```diff
@@ -73,17 +73,20 @@ class FlusherQueue {
 
 class Flusher : public WorkerBase {
 public:
+    enum FlusherType { GENERIC = 0x0, ONLY_HANDLE_ASYNC = 0x1, ONLY_HANDLE_SYNC = 0x2 };
```

Comment:
While I agree on `GENERIC` for non-dedicated situation, but the other two need to be renamed. Actually both are async processing. I'd rather name them
```c++
enum FlusherType { GENERIC = 0x0, FLUSH_ON_DEMAND = 0x1, FLUSH_ON_CONDITION = 0x2 };
```